hive-metastore/src/main/java/org/apache/iceberg/hive/HiveTableOperations.java

```diff
@@ -230,12 +190,14 @@ protected void doCommit(TableMetadata base, TableMetadata metadata) {
                 .collect(Collectors.toSet());
       }
 
-      Map<String, String> summary =
-          Optional.ofNullable(metadata.currentSnapshot())
-              .map(Snapshot::summary)
-              .orElseGet(ImmutableMap::of);
-      setHmsTableParameters(
-          newMetadataLocation, tbl, metadata, removedProps, hiveEngineEnabled, summary);
+      HMSTablePropertyHelper.updateHmsTableForIcebergView(
```

Comment:
This should be `updateHmsTableForIcebergTable`,

Reply:
Fixed.